docs/internal/log4net.md

```diff
@@ -0,0 +1,60 @@
+# `log4net` instrumentation
+
+> [!IMPORTANT]
+> log4net bridge and trace context injection are experimental features.
+> Both instrumentations can be disabled by setting `OTEL_DOTNET_AUTO_LOGS_LOG4NET_INSTRUMENTATION_ENABLED` to `false`.
```

Comment:
I don't see the env variable `OTEL_DOTNET_AUTO_LOGS_LOG4NET_INSTRUMENTATION_ENABLED` anywhere, only `OTEL_DOTNET_AUTO_LOGS_ENABLE_LOG4NET_BRIDGE`. Am I missing something?

Reply:
We have two levels
1. Injecting context to the log files by standard funtionality https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/pull/3825/files#diff-77b3519370ff48f6caf7c316f0ece966f2f0871c40e9381d451d2c5b1c76b40fR124 
2. Bridging everything to the exporter (typically OTLP) by `OTEL_DOTNET_AUTO_LOGS_ENABLE_LOG4NET_BRIDGE`

Reply:
`OTEL_DOTNET_AUTO_LOGS_LOG4NET_INSTRUMENTATION_ENABLED` is part of our standard functionality